app/controllers/tenants/projects_controller.rb

```diff
@@ -58,6 +58,7 @@ def project_params
                                     :requested_by,
                                     :retired, :tenant_id, :repayment_begin,
                                     :repayment_rate, categories: [])
+                                    # photos: {image: [:image_file_name, :image_content_type]}
```

Comment:
Line is too long. [94/80]<br>Incorrect indentation detected (column 36 instead of 4).